test/cmd/templates.sh

```diff
@@ -11,11 +11,13 @@ os::log::install_errexit
 # This test validates template commands
 
 oc get templates
-oc create -f examples/sample-app/application-template-dockerbuild.json
+oc create -f examples/sample-app/application-template-dockerbuild.json 
+oc create -f examples/sample-app/application-template-dockerbuild.json -n openshift
```

Comment:
@smarterclayton do you know if we run these as "admin" user? I need to create these in the namespace that the user cannot create processedtemplate but can view them :-)